### PR TITLE
tutorials: more blog fixes

### PR DESCRIPTION
**Pull request by Serkonda**
- Update markdown guide
- Fix redirect in new_article

### CODE REVIEW

File: tutorials/building-a-simple-web-blog-with-vweb.md [hunk 1]
```diff
@@ -170,48 +168,22 @@ into a single binary file together with the web application itself.
 
 - All errors in the templates are guaranteed to be caught during compilation.
 
+
 ### Fetching data with V ORM
 
 Now let's display some articles!
 
-We'll be using V's builtin ORM and a Postgres database. (V ORM will also
```
Comment (medvednikov) [inline tutorials/building-a-simple-web-blog-with-vweb.md:177]: No need to remove the upcoming DB support

File: tutorials/building-a-simple-web-blog-with-vweb.md [hunk 2]
```diff
@@ -221,7 +193,10 @@ Modify the `init_once()` method we created earlier to connect to a database:
 
 ```v
 pub fn (mut app App) init_once() {
-	db := sqlite.connect(':memory:') 	or { panic(err) }
+	db := sqlite.connect(':memory:') or { panic(err) }
```
Comment (medvednikov) [inline tutorials/building-a-simple-web-blog-with-vweb.md:196]: This should use a file:`blog.sqlite`

**Comment by tleen** [comment]
Funny I was working on this too. Do you think, for the ease of not dealing much with the database outside of the tutorial, that it create the db and initially populate it in the `init_once()` and do the commands inline?

```v
pub fn (mut app App) init_once() {
	// We will connect to our in-memory SQLLite instance and create our
	// article table with some data.
	db := sqlite.connect(':memory:') or {
		panic(err)
	}
	db.exec("create table article (id integer not null primary key, title text default '', text text default '')")
	db.exec("insert into article (title, text) values ('Hello, world!', 'V is great.'),('Second post.', 'Hm... what should I write about?')")
	app.db = db
}
```

This would keep the overall amount of stuff people have to do to get the tutorial to a minimum?

**Comment by Serkonda** [comment]
@tleen  To be honest I had no idea before this PR about ORM or databases in general but now I have a basic understanding.
In conclusion this tutorial brings insights into many topics without the need of deeper knowledge. 
I don't think this would be a bad thing.

File: tutorials/building-a-simple-web-blog-with-vweb.md [hunk 3]
```diff
@@ -170,48 +168,47 @@ into a single binary file together with the web application itself.
...
 
+
 ### Fetching data with V ORM
 
 Now let's display some articles!
 
-We'll be using V's builtin ORM and a Postgres database. (V ORM will also
-support MySQL, SQLite, and SQL Server soon.)
+We'll be using V's builtin ORM and a SQLite database. 
+(V ORM will also support MySQL, Postgre, and SQL Server soon.)
```
Comment (medvednikov) [inline tutorials/building-a-simple-web-blog-with-vweb.md:177]: Postgres

**Comment by medvednikov** [comment]
@tleen I just don't want test posts to disappear :)

Want this to behave like a real web app that saves your input.

**Pull request merged**